packages/flutter_test/lib/src/binding.dart

```diff
@@ -505,6 +505,17 @@ abstract class TestWidgetsFlutterBinding extends BindingBase
   /// then flushes microtasks.
   ///
   /// Set to null to use the default surface size.
+  ///
+  /// To avoid affecting other tests by leaking state, a test that
+  /// uses this method should always reset the surface size to the default.
+  /// For example, using `addTearDown`:
+  /// ```dart
+  ///   await binding.setSurfaceSize(someSize);
+  ///   addTearDown(() => binding.setSurfaceSize(null));
+  /// ```
+  ///
+  /// See also [TestFlutterView.physicalSize], which has a similar effect.
+  // TODO(pdblasi-google): Deprecate this. https://github.com/flutter/flutter/issues/123881
   Future<void> setSurfaceSize(Size? size) {
```

Comment:
I tried writing this cross-reference in the form "Consider using [TestFlutterView.physicalSize] instead, …", but realized I wasn't sure how to finish the sentence with a concise "because".

@pdblasi-google Happy to take suggestions, or we can just leave it for when you pick up #123881.

Reply:
Looks good to me, I appreciate the docs driving people to use `addTearDown`!